mappings.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
+<mappings>
+  <mapping>
+    <path>~/navigation/position/latitude</path>
+    <sentence>
+      <id>RMC</id>
+      <fieldset type="coordinate">
+        <field>2</field>
+        <field>3</field>
+      </fieldset>
+      <filter type="if">
+        <field>1</field>
+        <value transform="uppercase">A</value>
+      </filter>
+    </sentence>
+  </mapping>
+
+  <mapping>
+    <path>~/navigation/position/longitude</path>
```

Comment:
I was referring to the "description" parts of the delta.json schema.. Main issue was confusion, as I haven't used delta in a while!